assets/src/edit-story/elements/shared/index.js

```diff
@@ -42,7 +42,7 @@ export const elementWithPosition = css`
 
 export const elementWithSize = css`
   width: ${({ width }) => `${width}px`};
-  height: ${({ height }) => `${height}px`};
+  height: ${({ height }) => `${Math.round(height)}px`};
```

Comment:
@barklund This seems to fix the jumpiness caused by inconsistencies between edit-display mode, looks like it happens when height is adjusted due to some related value changing, e.g. font size / line-height which can cause a difference due to using partial vs full pixels. Thoughts?

Reply:
Note: this seems to also be necessary for https://github.com/google/web-stories-wp/pull/3125